training/src/main/java/wolox/training/models/Book.java

```diff
@@ -110,6 +118,7 @@ public Integer getPages() {
     }
 
     public void setPages(Integer pages) {
+        Preconditions.checkArgument(pages!=null && pages > 1, "The book must have at least 1 page");
```

Comment:
A tiny detail, if the book has one page this will throw exception, so you should change to "pages >= 1" to match the condition in the message 😅

Reply:
Good catch!